### PR TITLE
Rename Performance Addon Controller to Performance Profile Controller

### PR DESCRIPTION
**Pull request by marioferh**
Rename old names
Clean lower case and uniform all names

Signed-off-by: Mario Fernandez <mariofer@redhat.com>

### CODE REVIEW

File: docs/performanceprofile/api-versions.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 ## Supported API Versions
 
-The Performance Addon controller supports *v2*, *v1* and *v1alpha1* [Performance Profile](performance_profile.md) versions.
+The Performance Controller Operator supports *v2*, *v1* and *v1alpha1* [Performance Profile](performance_profile.md) versions.
```
Comment (fromanirh) [inline docs/performanceprofile/api-versions.md:5]: this is not a operator no more, so we should write "Performance Profile Controller" or something along these lines (here and below)

Comment (marioferh) [reply docs/performanceprofile/api-versions.md]: done

**Comment by fromanirh** [comment]
/lgtm
/hold
generally speaking this seems right, it's 4.12+ material, right?
hold to let other reviewers chime in

**Comment by marioferh** [comment]
> /lgtm /hold generally speaking this seems right, it's 4.12+ material, right? hold to let other reviewers chime in

Well the change has been doing in 4.11. It's not super important but I don't have a strong opinion about it.

**Comment by fromanirh** [comment]
It's a weird timing for this change to land in 4.11, but we can re-evaluate

**Comment by marioferh** [comment]
> It's a weird timing for this change to land in 4.11, but we can re-evaluate

yep then better for 4.12

**Comment by fromanirh** [comment]
/approve
/hold cancel
for 4.12/main branch only

**Pull request merged**